app/build.gradle

```diff
@@ -174,6 +173,10 @@ dependencies {
     implementation 'com.squareup.moshi:moshi:1.6.0'
     kapt 'com.squareup.moshi:moshi-kotlin-codegen:1.6.0'
 
+    // Navigation
+    implementation "android.arch.navigation:navigation-fragment-ktx:${navigation_version}"
```

Comment:
It looks like `2.0.0` is stable with AndroidX? Should we be using that instead?

Reply:
yes